internal/humanitec_go/types/deltas.go

```diff
@@ -13,6 +13,10 @@ type CreateDeploymentDeltaRequest struct {
 	Shared   []UpdateAction `json:"shared,omitempty"`
 }
 
+type UpdateDeploymentDeltaRequest struct {
```

Comment:
Deltas can also include shared resources.
There should be a `shared` property wich takes an array of JSON Patch fields scoped to the root `shared` parameter in the deployment set.

See `CreateDeploymentDeltaRequest` above. The object are the same in this case - so would suggest you just reuse the delta object?

Reply:
aha yes I missed that because the api docs were missing that in the example. fixed now 👍 
(going to keep the update struct without the id+metadata to reduce confusion on this end)